backend/sprint/model_project/models.py

```diff
@@ -1,44 +1,44 @@
 from django.db import models
-from django.contrib.auth.models import User
+from django.contrib.auth import get_user_model
```

Comment:
Is there any reason using `get_user_model` function instead of Django User?

Reply:
In pylint-django, it is not permitted to use auth.model.User directly.
I should change the code like that to pass travis CI - pylint seciton.